rust-1.81.yaml

```diff
@@ -10,27 +10,30 @@ package:
     memory: 16Gi
   dependencies:
     runtime:
-      - libLLVM-18
+      - libLLVM-${{vars.llvm-vers}}
     provides:
       - rust=${{package.full-version}}
 
+vars:
+  llvm-vers: 18
```

Comment:
```suggestion
  llvm-vers: 19
```

Reply:
Ahh yeah that would be important and rust doesn't build with llvm-19 it looks like : (.